tests/ObjectEncryptorTest.php

```diff
@@ -74,6 +74,7 @@ public function testEncryptorInvalidService()
             $encryptor->encrypt('secret', 'fooBar');
             self::fail('Invalid crypto wrapper must throw exception');
         } catch (ApplicationException $e) {
+            self::assertContains('Invalid crypto wrapper fooBar', $e->getMessage());
```

Comment:
$this->expectException() + $this->expectExceptionMessage()

Reply:
no, uz jsem to zacal delat, jenze to bych ty testy musel prepsat skoro vsechny, moje OCD me nedovoluje mit na jednom expcecException, na druhym try-catch a na tretim @expectEception :)

Reply:
radsi bych to udelal v separanim PR, at to neni vic splacnutych zmen, ten assert jsem musel doplnit kvuli phpunit7 (ktery uz nedovoluje spustit test bez assertu)

Reply:
Jo, v pohodě, proto jsem to taky dal jen jako comment a ne review. Nikdy nevím, co kdo zná, tak radši sypu všude všechno ;)

Reply:
https://github.com/keboola/object-encryptor/issues/7